ia/src/ml_agent.py

```
@@ -1,6 +1,7 @@
 import protocole
 import game
 import math
+import random
```

Pylint (3.3.7)

[convention] standard import "random" should be placed before third party imports "protocole", "game"

(C0411)